docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,153 @@
+
+- Access to an Aiven for Apache Kafka service with
+[Apache Kafka Connect](/docs/products/kafka/kafka-connect/howto/enable-connect)
+enabled.
+- Administrative access to a PostgreSQL database with SSL enabled.
+- The following SSL certificates and keys obtained from your PostgreSQL database:
+  - SSL client certificate: The public certificate for client authentication.
+  - SSL root certificate: The certificate of the Certificate Authority (CA) that signed
+  the servers' and clients' certificates.
+  - SSL client key: The client's private key is used to encrypt the data sent to the
+  server.
+  For additional details, see
+  [Certificate requirements](/docs/platform/concepts/tls-ssl-certificates#certificate-requirements).
+
+## Variables
+
+The following table lists the variables for the configuration steps. Replace them with
+your actual environment values in the provided code snippets:
+
+| Variable | Description |
+|---|---|
+| `<kafka_cluster_name>` | Name of your Aiven for Apache Kafka service |
+| `<kafka_connect_name>` | Name of your Apache Kafka Connect service |
+| `<cloud_provider_and_region>` | Your cloud provider and region identifier |
+| `<plan>` | Service plan for Apache Kafka/Apache Kafka Connect |
+| `<project_name>` | Name of your project in the Aiven platform |
+| `<integration_config>` | JSON configuration for PostgreSQL endpoint |
+| `<integration_endpoint_id>` | ID of the created PostgreSQL endpoint |
+| `<connector_config>` | JSON configuration for Debezium connector |
+
+## Configuration via CLI
+
+1. If you already have an Aiven for Apache Kafka service, make sure it's running and
+accessible. Otherwise, create a new Kafka cluster using this command:
+
+   ```bash
+   avn service create <kafka_cluster_name> \
+   --service-type kafka \
+   --cloud <cloud_provider_and_region> \
+   --plan <plan> \
+   --project <project_name>
+   ```
+
+1. Create an Apache Kafka Connect service and configure it to communicate with your
+Aiven for Apache Kafka service:
+
+   ```bash
+   avn service create <kafka_connect_name> --service-type kafka_connect \
+   --cloud <cloud_provider_and_region> --plan <plan> --project $PROJECT
+   ```
+
+1. Create an external PostgreSQL integration endpoint to represent your PostgreSQL
+database. Use the following JSON configuration and replace the placeholders:
+
+   ```bash
+   INTEGRATION_CONFIG=$(cat <<-END
+   {
+    "ssl_mode": "verify-ca",
+    "host": "<postgresql_host>",
+    "port": <postgresql_port>,
+    "user": "<postgresql_user>",
+    "ssl_client_certificate": "$(cat /path/to/your/client-cert.pem)",
+    "ssl_root_cert": "$(cat /path/to/your/ca.pem)",
+    "ssl_client_key": "$(cat /path/to/your/client-key.pem)"
+   }
+   END
+   )
+
+   avn service integration-endpoint-create --project $PROJECT -d external_postgresql \
+   --user-config "$INTEGRATION_CONFIG"
+
+   ```
+
+1. Retrieve endpoint ID of the integration endpoint you just created using this command:
```

Comment:
oh the ordering seems wrong, from here on its always 1

Reply:
This is intentional. In our documentation setup, we maintain consistency by starting all numbered list items with 1 to facilitate easy insertion of steps. The numbering is adjusted during output generation.